src/main/java/io/kestra/plugin/aws/runner/AwsBatchScriptRunner.java

```diff
@@ -154,28 +153,21 @@ public class AwsBatchScriptRunner extends ScriptRunner implements AbstractS3, Ab
         title = "The maximum duration to wait for the job completion. AWS Batch will automatically timeout the job upon reaching such duration and the task will be failed."
     )
     @Builder.Default
-    private final Duration waitUntilCompletion = Duration.ofHours(1);
+    private Duration waitUntilCompletion = Duration.ofHours(1);
+
+    @JsonIgnore
+    @Getter(AccessLevel.NONE)
+    private Path batchWorkingDirectory;
+
+    @JsonIgnore
+    @Getter(AccessLevel.NONE)
+    private Path batchOutputDirectory;
```

Comment:
I' not a fan of having attributes that are not coming from script runner deserialization.
If there is no other way, at least, mark them as `transient` so it's clear that it's not coming from deserializatin.